packit/schema.py

```diff
@@ -361,28 +361,28 @@ class JobMetadataSchema(Schema):
           dist_git_branch deprecation period.
     """
 
-    _targets = Targets(missing=None, data_key="targets")
+    _targets = Targets(load_default=None, data_key="targets")
     timeout = fields.Integer()
-    owner = fields.String(missing=None)
-    project = fields.String(missing=None)
-    dist_git_branches = DistGitBranches(missing=None)
-    branch = fields.String(missing=None)
+    owner = fields.String(load_default=None)
+    project = fields.String(load_default=None)
+    dist_git_branches = DistGitBranches(load_default=None)
+    branch = fields.String(load_default=None)
     scratch = fields.Boolean()
     list_on_homepage = fields.Boolean()
     preserve_project = fields.Boolean()
     use_internal_tf = fields.Boolean()
-    additional_packages = fields.List(fields.String(), missing=None)
-    additional_repos = fields.List(fields.String(), missing=None)
-    bootstrap = EnumField(MockBootstrapSetup, missing=None)
-    fmf_url = fields.String(missing=None)
-    fmf_ref = fields.String(missing=None)
-    fmf_path = fields.String(missing=None)
+    additional_packages = fields.List(fields.String(), load_default=None)
+    additional_repos = fields.List(fields.String(), load_default=None)
+    bootstrap = fields.Enum(MockBootstrapSetup, load_default=None)
```

Comment:
```suggestion
    bootstrap = EnumField(MockBootstrapSetup, load_default=None)
```

shouldn't this also use the compatibility shim?

Reply:
Oops, yes. That was a leftover from debugging.